Vizzuality/marxan-cloud
# Feature/refresh token

### Overview

**Don't merge before #89**

This PR solves an issue where API throws 401 when the session has been expired in the API but not in the client. The approach followed is: when the token has been expired we will refresh the token automatically. We will only sign out when the user clicks on Sign out.

### Testing instructions

1. Sing in with a valid user.
2. Wait for 2 hours :sweat_smile: I don't sure If there is any other way to check this...
3. The token will be requested again and it should work again with a new token. So, the session shouldn't be expired.
4. Sign-out. The token will be expired.

### Feature relevant tickets

[MARXAN-121](https://vizzuality.atlassian.net/browse/MARXAN-121?atlOrigin=eyJpIjoiN2FlMjg4NmJiNTEzNDU2ZjgwM2ZkYjJmODgwZGNmNzkiLCJwIjoiaiJ9)
